superflore/generators/ebuild/ebuild.py

```diff
@@ -215,7 +215,7 @@ def get_ebuild_text(self, distributor, license_text):
             ret += "    " + 'ros-{0}/{1}\n'.format(self.distro, bdep)
         for bdep in sorted(self.depends_external):
             try:
-                for res in resolve_dep(bdep, 'gentoo')[0]:
+                for res in resolve_dep(bdep, 'gentoo', self.distro)[0]:
```

Comment:
Should adding the distro argument be a separate PR? I think that can be isolated from this change set.

Reply:
I've moved that to https://github.com/ros-infrastructure/superflore/pull/173 . It would be nice if you could help on verifying that Gentoo also works well after passing distro to resolve_rosdep_key.